zetaclient/chains/bitcoin/observer/inbound_test.go

```diff
@@ -13,30 +13,39 @@ import (
 	"github.com/btcsuite/btcd/chaincfg"
 	"github.com/btcsuite/btcd/wire"
 	"github.com/btcsuite/btcutil"
+	"github.com/pkg/errors"
 	"github.com/rs/zerolog/log"
+	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
 
 	"github.com/zeta-chain/node/pkg/chains"
 	"github.com/zeta-chain/node/zetaclient/chains/bitcoin"
 	"github.com/zeta-chain/node/zetaclient/chains/bitcoin/observer"
+	"github.com/zeta-chain/node/zetaclient/chains/interfaces"
 	clientcommon "github.com/zeta-chain/node/zetaclient/common"
 	"github.com/zeta-chain/node/zetaclient/testutils"
 	"github.com/zeta-chain/node/zetaclient/testutils/mocks"
 )
 
-// createRPCClientAndLoadTx is a helper function to load raw tx and feed it to mock rpc client
-func createRPCClientAndLoadTx(t *testing.T, chainId int64, txHash string) *mocks.MockBTCRPCClient {
-	// file name for the archived MsgTx
-	nameMsgTx := path.Join(TestDataDir, testutils.TestDataPathBTC, testutils.FileNameBTCMsgTx(chainId, txHash))
+// createRPCClientAndLoadTx is a helper function to load raw txs and feed them to mock rpc client
+func createRPCClientAndLoadTx(t *testing.T, chainId int64, txHashes ...string) interfaces.BTCRPCClient {
```

Comment:
Consider moving this function to the testutils package

Reply:
good point. Let me try it.